### PR TITLE
UX: Update Submissions Admin UI

### PR DESCRIPTION
**Pull request by keegangeorge**
### Incorporate Discourse UI conventions to admin panel:
- Add Wizard Message Component to each pane
- Improve submission data and showcase each unique to type
- Add feature to edit columns in submissions pane

### CODE REVIEW

File: assets/javascripts/discourse/controllers/admin-wizards-submissions-show.js.es6 [hunk 1]
```diff
@@ -24,6 +27,21 @@ export default Controller.extend({
...
       });
   },
+  
+  
+  @discourseComputed('submissions', 'fields.@each.enabled')
+  displaySubmissions(submissions, fields) {
+    return submissions.map(submission => {
+      let field = fields.find(f => Object.keys(submission).includes(f.id));
+      if (!field.enabled) {
+        // insert field / submission deletion code here:
```
Comment (keegangeorge) [inline assets/javascripts/discourse/controllers/admin-wizards-submissions-show.js.es6:37]: Here is where I am having some issues, as discussed in our PM.

Comment (angusmcleod) [reply assets/javascripts/discourse/controllers/admin-wizards-submissions-show.js.es6:37]: Perhaps we could use a reduce here?

**Comment by keegangeorge** [comment]
@angusmcleod 

Not sure why but getting a linting error with prettier, seems to be related to the license?:
```zsh
warning package.json: License should be a valid SPDX license expression
```

**Comment by angusmcleod** [comment]
@keegangeorge The errors are coming from these files. Probably need to run prettier on them again.

Screenshot at Sep 02 14-50-14

**Comment by keegangeorge** [comment]
@angusmcleod 
✅ All tests have passed! 😃

**Pull request merged**